ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/versions/electra/BeaconBlockBodyElectra.java

```diff
@@ -32,6 +32,11 @@ static BeaconBlockBodyElectra required(final BeaconBlockBody body) {
 
   ExecutionRequests getExecutionRequests();
 
+  @Override
+  default Optional<ExecutionRequests> getOptionalExecutionRequests() {
```

Comment:
I guess we can get rid of the `ExecutionRequests getExecutionRequests();` and move this to the `BlindedBeaconBlockBodyElectraImpl`. WDYT?

Reply:
humm.. this way is consistent with the other fork. For now I'd leave as it is.